appengine/pond/duck/rank/js/rank.js

```diff
@@ -16,44 +16,84 @@
  */
 
 /**
- * @fileoverview Creates a view of all the ducks for a user.
- * @author aschmiedt@google.com (Abby Schmiedt)
+ * @fileoverview Computes leaderboard ranking matches.
  */
 'use strict';
 
 goog.provide('Pond.Duck.Rank');
 
+goog.require('Pond.Avatar');
+goog.require('Pond.Battle');
+goog.require('Pond.Duck');
 goog.require('Pond.Duck.Datastore');
 
+Pond.Duck.Rank.MATCH_KEY = null;
+Pond.Duck.Rank.RANK_ENTRIES = [];
+
 /**
- * .
+ * Initialize interpreter and triggers match request. Called on page load.
  */
 Pond.Duck.Rank.init = function () {
```

Comment:
What about RankingBattle? Since it is still running a battle it is just a specific kind of battle that is meant to tell the ranking?

Reply:
I think my main hesitation is that rankingBattle is a long name, while the other files have stuck to one word names. But also, Battle is the game logic and this file is more similar to Pond or Duck which run Battles.

Reply:
Discussed offline, will address refactoring Rank in a different PR.

Reply:
Discussed offline, will address refactoring Rank in a different PR.